samcli/commands/local/cli_common/click_mutex.py

```diff
@@ -41,7 +41,7 @@ def handle_parse_result(self, ctx, opts, args):
                     if mutex_opt in opts:
                         req_cnt -= 1
 
-                if req_cnt == 0:
+                if not req_cnt:
```

Comment:
Does it complain about this? It's weird using boolean logic on integers.

Reply:
In pylint's source

> Most of the times you should use the fact that integers with a value of 0 are false.
    An exception to this rule is when 0 is allowed in the program and has a
    different meaning than None!

https://github.com/PyCQA/pylint/blob/master/pylint/extensions/comparetozero.py#L23